build.rs

```diff
@@ -16,6 +16,7 @@ fn main() {
             build.flag("-D");
             build.flag("USE_KEXT");
             build.include("c_src/Karabiner-VirtualHIDDevice/dist/include");
+            build.include("c_src/Karabiner-DriverKit-VirtualHIDDevice/src/Client/vendor/include");
```

Comment:
I don't think this include is needed for `kext`. Can you confirm you can't compile without it?
Maybe `driver_activated` is what required it, and now that it's behind the `USE_KEXT` flag this is no longer needed?
If you can't compile without it, it means other parts of the code need to be behind the `USE_KEXT` flag too.